lib/checks/color/color-contrast-evaluate.js

```diff
@@ -40,14 +40,33 @@ export default function colorContrastEvaluate(node, options, virtualNode) {
     return undefined;
   }
 
+  const nodeStyle = window.getComputedStyle(node);
+  const fontSize = parseFloat(nodeStyle.getPropertyValue('font-size'));
+  const fontWeight = nodeStyle.getPropertyValue('font-weight');
+  const bold = parseFloat(fontWeight) >= boldValue || fontWeight === 'bold';
+
+  const ptSize = Math.ceil(fontSize * 72) / 96;
```

Comment:
I don't love these magic numbers, especially since they're now duplicated. Thoughts on hoisting them out into constants somewhere?

Reply:
I only moved code, lets not overcomplicate the changes